### DISCUSSION TITLE
Is the GitHub Student Pack really for students?

### DISCUSSION THREAD

**Discussion by thesumedh** [community/community#111307] [GitHub Education]
### Select Topic Area

Show & Tell

### Body

I am a student currently pursuing a four year degree course in a Tier 3 college. My college doesn't have the infrastructure to provide email addresses to each of their students. Although GitHub says that they accept dated ID cards, I have been unsuccessful in getting approved despite providing the same. I have been trying for days now, and have spent hours today and uploaded every possible document including all my fees receipts, semester grade cards and dated ID card. GitHub just apologizes on submission and rejects my application without any proper reason or way to fix it. There are multiple community threads with people facing similar issues. Is there any other way for Tier 3 college students to verify themselves without university email addresses?
Is GitHub Student pack is really for student?
I'm trying since 6 months and rejected 10 times. It's very sad they are not fixed this... 

![Screenshot 2024-03-08 150742](https://github.com/community/community/assets/118736444/4640e6b0-c478-4785-89f7-400579bfd2c3)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @thesumedh ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
